output/cloud/expv2/integration/testdata/metricset.json

```diff
@@ -119,7 +119,23 @@
           "trendHdrSamples": {
             "values": [
               {
-                "time": "2023-05-01T04:00:01.500Z"
+                "time": "2023-05-01T04:00:01.500Z",
+                "count": 1,
+                "counters": [
+                  0,
+                  0,
+                  0,
+                  0,
+                  0,
+                  0,
+                  1
```

Comment:
auto spot as a reminder, now with the aggregation we need to cover the trimzeros from `addToBucket`